libraries/WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -36,6 +36,12 @@ class FunctionRequestHandler : public RequestHandler {
 
         return true;
     }
+    bool canRaw(String requestUri) override {
+        if (!_ufn || _method == HTTP_GET)
+            return false;
```

Comment:
```suggestion
        if (!_ufn || _method == HTTP_GET) {
            return false;
        }
```

Reply:
Why ? The rest of the code does not have this formatting.